docs/source/commandline/commandline-all.rst

```diff
@@ -1209,18 +1207,21 @@ Core Video Options
 
     Controls initial window size in windowed mode.  If it is set on, the window
     will initially be set to the maximum supported size when you start MAME.  If
-    it is turned off, the window will start out at the smallest supported size.
-    This option only has an effect when the -window option is used.
+    it is turned off, the window will start out at the closest possible size to
+    the original size of the display; it will scale on only one axis where
```

Comment:
This one's escaping me at the moment because it's difficult to find a phrasing that won't go too deep into technical terms. I'll come back to this one after I have a chance to talk to you.